tests/integration/solidity_contracts/PlainOpcodes/test_safe.py

```diff
@@ -29,3 +29,12 @@ async def test_should_withdraw_call_eth(self, safe, owner, eth):
             assert (
                 await eth.balanceOf(owner.starknet_address).call()
             ).result.balance.low == ACCOUNT_BALANCE
+
+    class TestDeploySafeWithValue:
+        async def test_deploy_safe_with_value(
+            self, safe, safe_deployer, deploy_solidity_contract
+        ):
+            safe = await deploy_solidity_contract(
+                "PlainOpcodes", "Safe", value=int("1", 16)
```

Comment:
please create a `value = 1`, and use it in other places in the test

Reply:
done [here](https://github.com/kkrt-labs/kakarot/pull/725/commits/8bd81fe69a5f12d049c864cf98707e29c970e99b).